src/components/MessagesList/MessagesGroup/Message/Message.vue

```diff
@@ -451,6 +472,10 @@ export default {
 			return moment(this.timestamp * 1000).format('LL')
 		},
 
+		messageDateTime() {
+			return moment(this.timestamp * 1000).format('llll')
+		},
```

Comment:
A long format makes this item the longest item in the message bar. Maybe week day and use only `lll`?

Reply:
@nickvergessen